pyqtgraph/__init__.py

```diff
@@ -85,6 +84,19 @@ def getConfigOption(opt):
     return CONFIG_OPTIONS[opt]
 
 
+
+
+
+### Set style options
+## All options related to style are set here
+## Init default style
+from .style.core import loadDefautStyle
```

Comment:
should that be "DefaultStyle" throughout, maybe?

Reply:
Spelling: "Defaut" != "Default"

Reply:
lol that's a very French mistake to do ^^ .

Thanks.